Moovies/Moovies/SceneDelegate.swift

```diff
@@ -12,14 +12,18 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         options connectionOptions: UIScene.ConnectionOptions
     ) {
         guard let _ = (scene as? UIWindowScene) else { return }
-        let vc = MooviesViewController()
-        let films = Film(results: [], totalResults: 0, totalPages: 0, page: 0)
-        let presenter = MainPresentor(view: vc, model: films)
-        vc.presentor = presenter
-        let navigationController = UINavigationController(rootViewController: vc)
-        window?.rootViewController = navigationController
-        navigationController.navigationBar.backgroundColor = .systemPink
+
+        let mainVC = ModulesBuilder.buildMain()
+        window?.rootViewController = mainVC
         window?.makeKeyAndVisible()
+//        let vc = MooviesViewController()
```

Comment:
```suggestion
//        let vc = MooviesViewController()
```
удали весь закоменченый